src/msg.c

```diff
@@ -777,51 +781,55 @@ void msg_copy_to_print_buffer(MessagePrintState* printer, s32 arg1, s32 arg2) {
                         printer->windowSize.x = *srcBuf++;
                         printer->windowSize.y = *srcBuf++;
                         sfx_play_sound_with_params(SOUND_21C, 0, 0, 0);
-                        printer->windowState = 2;
+                        printer->windowState = MSG_WINDOW_STATE_OPENING;
                         printer->delayFlags |= 1;
-                        printer->stateFlags |= 0x800;
+                        printer->stateFlags |= MSG_STATE_FLAG_800;
                         break;
                     case MSG_STYLE_LAMPPOST:
                         printer->windowSize.y = *srcBuf++;
                         /* fallthrough */
                     case MSG_STYLE_SIGN:
-                        if (!s8) {
-                            printer->windowState = 2;
-                            printer->stateFlags |= 0x800;
-                            printer->delayFlags |= 1;
-                        }
+                        do {
+                            if (!s8) {
+                                printer->windowState = MSG_WINDOW_STATE_OPENING;
+                                printer->stateFlags |= MSG_STATE_FLAG_800;
+                                printer->delayFlags |= 1;
+                            }
+                        } while (0);
                         break;
                     case MSG_STYLE_POSTCARD:
                         arg = *srcBuf++;
-                        printer->windowState = 2;
-                        printer->stateFlags |= 0x800;
+                        printer->windowState = MSG_WINDOW_STATE_OPENING;
+                        printer->stateFlags |= MSG_STATE_FLAG_800;
                         printer->delayFlags |= 1;
                         printer->letterBackgroundImg = heap_malloc(7875);
```

Comment:
This is sizeX * sizeY / 2 for postcard, perhaps include the header and use those?